index.html

```diff
@@ -1,390 +1,445 @@
-                </ion-icon>
+                <ion-icon name="chevron-forward-outline"> </ion-icon>
               </li>
             </ul>
-            <hr>
+            <hr />
           </div>
         </div>
+      </section>
+    </main>
+    <footer>
+      <div class="footer-text">
+        <h2 class="title">Contact me</h2>
+        <p class="supporting-text">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
       </div>
-    </section>
-
-  </main>
-  <footer>
-    <div class="footer-text">
-      <h2 class="title">Contact me</h2>
-      <p class="supporting-text">If you have an
-        application you are interested in developing,
-        a feature that you need built or a project that
-        needs
-        coding. I’d love to help with it</p>
-    </div>
-    <form action="https://formspree.io/f/xwkyvenp" method="post" class="form">
-      <div class="name">
-        <label for="name"></label>
-        <input type="text" name="full-name" id="name"
-          placeholder="Full name" required minlength="2"
-          maxlength="30">
-      </div>
-      <div class="email">
-        <label for="mail"></label>
-        <input type="email" name="mail" id="mail"
-          placeholder="Email"
-          pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$"
-          required>
-      </div>
-      <div class="message">
-        <label for="message"></label>
-        <textarea name="message" id="message"
-          placeholder="Enter text here" required
-          maxlength="500" minlength="10"></textarea>
-      </div>
-
-      <button type="submit" class="form-submit">Get
-        in
-        touch</button>
-    </form>
-
-
-    <img src="./images/footer-gb/Light.svg" alt="indicator">
-  </footer>
+      <form action="https://formspree.io/f/xwkyvenp" method="post" class="form">
+        <div class="name">
+          <label for="name"></label>
+          <input
+            type="text"
+            name="full-name"
+            id="name"
+            placeholder="Full name"
+            required
+            minlength="2"
+            maxlength="30"
+          />
+        </div>
+        <div class="email">
+          <label for="mail"></label>
+          <input
+            type="email"
+            name="mail"
+            id="mail"
+            placeholder="Email"
+            pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$"
+            required
+          />
+        </div>
+        <div class="message">
+          <label for="message"></label>
+          <textarea
+            name="message"
+            id="message"
+            placeholder="Enter text here"
+            required
+            maxlength="500"
+            minlength="10"
+          ></textarea>
+        </div>
 
+        <button type="submit" class="form-submit">Get in touch</button>
+      </form>
```

Comment:
- [ ] Please, consider, that you still have not fixed my mentioned issue with the "Get in touch" button. 
![2022-01-14 03 23 52](https://user-images.githubusercontent.com/49064106/149424512-de46a02c-7661-4415-ba15-a10e10b6094f.gif)